cdc/metrics_owner.go

```diff
@@ -44,6 +44,13 @@ var (
 			Name:      "maintain_table_num",
 			Help:      "number of replicated tables maintained in owner",
 		}, []string{"changefeed", "capture", "type"})
+	changefeedStatusGauge = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: "ticdc",
+			Subsystem: "owner",
+			Name:      "status",
+			Help:      "changefeed status",
+		}, []string{"feedstate"})
```

Comment:
```suggestion
			Help:      "The status of changefeeds",
		}, []string{"changefeed", "state"})
```

We can record changefeed name and status in the gauge.